deployment/lab/.env

```diff
@@ -0,0 +1,6 @@
+STAGE=lab
+DOMAIN_NAME=runsidekick.me
+SANDBOX_APP_DOMAIN_NAME=todoapp.sandbox.runsidekick.me
```

Comment:
it should be `todoapp.sandbox`

Reply:
eb-option-settings prepared to work with this config